packages/zero-protocol/src/query-hash.ts

```diff
@@ -13,3 +13,11 @@ export function hashOfAST(ast: AST): string {
   hashCache.set(normalized, hash);
   return hash;
 }
+
+export function hashOfNameAndArgs(
+  name: string,
+  args: readonly unknown[],
```

Comment:
can we not type args more specifically?

Reply:
good catch. Yes, it is `readonly ReadOnlyJSONValue[]` everywhere else.